MAVProxy/modules/mavproxy_link.py

```diff
@@ -147,6 +161,11 @@ def link_add(self, device):
         conn.last_heartbeat = 0
         conn.last_message = 0
         conn.highest_msec = 0
+        for attr in optional_attributes:
+            if not hasattr(conn, attr):
```

Comment:
There might be a reason you would want to do this... possibly to do something funky like set target_system here on a per link basis. However to do this we would need to convert the 'value' string into the correct type (int in the case of target_system). This could be done by passing in a single dict of key, value pairs and parsing the dict... /dev/ttyUSB0:{"target_system":123,"label":"link_for_system_123"}